### DISCUSSION TITLE
Tailwind changed everything for me.

### DISCUSSION THREAD

**Discussion by joshmanders** [tailwindlabs/tailwindcss#3217] [Kind words]
I've mentioned to you on Twitter many times my gratitude to you and Tailwind. But I want to reiterate it here... Tailwind CSS and Tailwind UI has literally changed EVERYTHING for me.

I've always had a good eye for design but my weakest thing was CSS... And no matter how much I tried, I just hated trying to get better at it.

When I saw you doing your KiteTail live streams and showed iterating fast with your LESS CSS framework, I was instantly hooked and kept pestering you on stream to release it for us to use. When you finally released v0.1 of Tailwind I was absolutely stunned and blown away by how great it was... I knew from that moment on I would never need anything else for styling my stuff.

Then with Tailwind UI, I knew it was gonna be so great that I literally held off building my own stuff for 3-6 months waiting for the release, and when you did, I bought 3 licenses (one for myself, one for a friend as a gift, and one to give away to someone)

You could make Tailwind UI license fee an annual thing and I wouldn't even second guess paying it. You've saved me so much time and money it's worth it.

I owe you so much for what you've done for me by releasing Tailwind CSS and Tailwind UI.

From the absolute bottom of my heart, Adam, Steve and contributors. Thank you so much.

Here's to a wonderful 2021 in Tailwind land.

**Comment by m1guelpf** [comment]
+1 on this, Tailwind actually taught me CSS (I learned the classes from the docs and then slowly learned the CSS properties they applied. The only reason I'm able to design cool stuff today is because of all the work you and the rest of the team has done. Thank you ❤️

**Reply by joshmanders** [reply]
Same, using Tailwind has actually improved my CSS skills in general. If I was ever working in a project that didn't use Tailwind and I wanted to say get rounded borders, I'd just go to Tailwind docs for `rounded` and copy the properties.

**Comment by simonswiss** [comment]
Thanks so much for sharing this @joshmanders - this is amazing! ❤️🤗
